src/IO/H5/CombineH5.hpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#pragma once
+
+#include <boost/program_options.hpp>
+#include <cstddef>
+#include <cstdlib>
+#include <iterator>
+#include <string>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/CheckH5PropertiesMatch.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/SourceArchive.hpp"
+#include "IO/H5/VolumeData.hpp"
+#include "Parallel/Printf.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "Utilities/MakeString.hpp"
+#include "Utilities/StdHelpers.hpp"
```

Comment:
You don't need any of these includes, only `<string>`. Move the rest to the .cpp if needed.

Reply:
yep